setup.cfg

```diff
@@ -54,6 +54,7 @@ tests_require =
     pytest-localserver==0.5.0
     pytest-mock==2.0.0
     pytest-benchmark==3.2.2
+    pytest-otel>=0.0.4
```

Comment:
Can you also add this to https://github.com/elastic/apm-agent-python/blob/master/tests/requirements/reqs-base.txt? That ensures that it is part of the docker build